### PR TITLE
Doc : added scholarships resources

### PR DESCRIPTION
**Pull request by ZeeshanMukhtar1**
## Assigned Issue

This issue  relates to #39

## Description

I have created a new file named "scholarships. md" in the "resources" folder to provide a valuable resource for students seeking financial aid for online courses. This file contains a link to my GitHub repository, which offers a comprehensive guide on how to apply for financial aid on Coursera. The resource aims to support learners in their educational endeavors.

## Type of PR

- [ ] ✨ Feature
- [ ] 🐞 Bug fix
- [ ] 📄 Documentation update
- [x] Others

This PR falls under the "Others" category as it involves adding a new resource to the project's collection.

### CODE REVIEW

File: resources/scholarships.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Supportive Tech Organizations
```
Comment (adiati98) [inline resources/scholarships.md:1]: Please change the title to "Tech Scholarships".

File: resources/scholarships.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Discover valuable resources to support your educational journey and access financial aid for online courses:
+
+- **[Coursera Financial Aid Guide](https://github.com/ZeeshanMukhtar1/Coursera-Financial-Aid-Application)**: A comprehensive guide on securing financial aid for Coursera courses. Created and maintained by [Zeeshan Mukhtar](https://www.linkedin.com/in/zeeshanmukhtar1/).
```
Comment (adiati98) [inline resources/scholarships.md:5]:
```suggestion
- **[Coursera Financial Aid Guide](https://github.com/ZeeshanMukhtar1/Coursera-Financial-Aid-Application)**: A comprehensive guide on securing financial aid for Coursera courses.
```

We don't need to mention who created and maintained here. Folks will know right away once they visit the link. 🙂

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @ZeeshanMukhtar1,
Thank you for the PR! 

I left some comments here for you to fix :)

**Comment by ZeeshanMukhtar1** [comment]
Hi @adiati98 , 
I've made the necessary corrections based on your feedback in the commit/PR. Kindly take another look when you have the chance.

Also, a big thank you for your guidance and assistance in our previous conversation; it has been invaluable in my open-source journey! 😊

**Review by adiati98 — APPROVED** [review]
Thank you for the changes, @ZeeshanMukhtar1! 🙌

git merge GIF

**Comment by adiati98** [comment]
> Hi @adiati98 , I've made the necessary corrections based on your feedback in the commit/PR. Kindly take another look when you have the chance.
> 
> Also, a big thank you for your guidance and assistance in our previous conversation; it has been invaluable in my open-source journey! 😊

You're most welcome, @ZeeshanMukhtar1! 😊

**Comment by ZeeshanMukhtar1** [comment]
Thank you for merging the PR, and I appreciate the opportunity to learn more about open source through this process. Your guidance has been invaluable

**Pull request merged**